spec/integration/make/dependencies_spec.rb

```diff
@@ -1,26 +1,30 @@
 require "spec_helper"
 
 RSpec.describe "Make dependencies" do
+  # rubocop:disable Lint/ConstantDefinitionInBlock
   # Some dependencies are only needed by the dependant app for their static files.
-  dependencies_that_do_not_need_to_be_running = %w[govuk-content-schemas]
+  DEPENDENCIES_THAT_DO_NOT_NEED_TO_BE_RUNNING = %w[govuk-content-schemas].freeze
```

Comment:
Sorry I'm late reviewing this. I found a slighter tidier approach if you want to review: https://github.com/alphagov/govuk-docker/pull/505